ush/rrfs_radmon/clean_tankdir.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
```

Comment:
Should this clean work be merged with the rrfs-workflow clean task?

Reply:
I prefer it stay in the radmon directory separately.